python_modules/dagster-graphql/dagster_graphql/schema/schedules/schedules.py

```diff
@@ -130,6 +134,26 @@ def resolve_futureTicks(self, _graphene_info, **kwargs):
         new_cursor = tick_times[-1] + 1 if tick_times else cursor
         return GrapheneFutureInstigationTicks(results=future_ticks, cursor=new_cursor)
 
+    def resolve_pastTicks(self, _graphene_info, **kwargs):
+        cursor = kwargs.get(
+            "cursor", get_timestamp_from_utc_datetime(get_current_datetime_in_utc())
+        )
+        tick_times = []
+        time_iter = self._external_schedule.execution_time_iterator(cursor, is_forward=False)
+
+        limit = kwargs.get("limit", 10)
+
+        for _ in range(limit):
+            tick_times.append(next(time_iter).timestamp())
+
+        past_ticks = [
+            GrapheneFutureInstigationTick(self._schedule_state, tick_time)
```

Comment:
Mostly disconcerted that we can return future ticks in the past.

some suggestions in random order of seriousness:

- UnfulfilledTick
- PotentialTick
- TickEvaluation
- UnrealizedTick
- ProtoTick
- TickPromise

Reply:
Went with PotentialTick: https://github.com/dagster-io/dagster/pull/11904